### ISSUE TITLE
UNDERTALE SPANISH TRANSLATION - Game not recognizing text functions (text color and character "emotes")

### ISSUE THREAD

**Issue by berrisfueler** [crumblingstatue/gmktool#1]
Unpacking and repacking works fine, but as I start the dialogue with Flowey, it doesn't seem to recognize the text color functions anymore, same with the "emote" ones.

Here's a link of my STRG.txt file: https://dl.dropboxusercontent.com/u/32075613/traduccionsp.tx

Here's how it looks in game:
![why](https://cloud.githubusercontent.com/assets/10138138/10868297/c93fb8aa-804f-11e5-8e3f-b849108d4d2f.png)

Thanks in advance!

**Comment by crumblingstatue** [comment]
I compared your text file to one freshly extracted from data.win, and it seems it has double the backslashes than normal.

Example:
```
Original:
\W* Hola^2!&* Soy\Y FLOWEY\W.^2 &* \YFLOWEY\W la \YFLOR\W!/
Yours:
\\W* Hola^2!&* Soy\\Y FLOWEY\\W.^2 &* \\YFLOWEY\\W la \\YFLOR\\W!/
```

Are double backslashes present even in a freshly extracted text file, or does this only happen after you save it with your editor. Maybe try a different editor. (Or did you put the double backslashes there manually?)

**Comment by berrisfueler** [comment]
Ohh, the first string dump I had I got it using quickbms, I tried dumping the strings again with QuickBMS and it added some extra backslashes like you said, then I tried dumping them with gmktool and the double backslashes were gone!

Now the color functions work! I feel kind of dumb for not re-dumping the files with gmktool first, thanks for the help!
![aww yiss](https://cloud.githubusercontent.com/assets/10138138/10869932/4e9f7a02-8083-11e5-9bf0-3edf2ff8d9bd.png)
